### DISCUSSION TITLE
Duda caso en que tienda esté llena

### DISCUSSION THREAD

**Discussion by IgnacioHqz** [IIC2133-PUC/2024-2#13] [Tarea 0] [answered]
Hola! 
tengo una duda en el caso de que la tienda esté llena. Entiendo que si está llena, de igual manera se tiene que leer los puffles del pingüino, pero la duda que tengo es que el pingüino servirá a futuro en algo? es decir, lo debería igualmente guardar para después usarlo o básicamente ese pingüino ya no servirá para nada más? No se si se entienda mi duda
Gracias!

**Comment by 4n1t4** [comment, marked as answer]
Si se entiende, no te preocupes por guardar el pingüino no es necesario.

**Reply by IgnacioHqz** [reply]
Perfecto! Muchas gracias!
